proposals/0003.md

```diff
@@ -0,0 +1,83 @@
+* Title: *Licensing and DCO*
+* Champion: *@FintanH/fintan.halpenny@gmail.com@*
+* Conversation:
+    * https://radicle.community/t/radicle-licensing-model/282
+    * https://radicle.community/t/were-implementing-a-developer-certificate-of-origin-dco/303
+* References:
+    * [Linux Kernel DCO][dco]
+    * [GPLv3 License][gplv3]
+
+# Summary
+
+The decision here is to implement our software under the license of
+GPLv3+linking exception. This also means that outside contributors must sign off
+on their commits to our software, as described by our Developer Certificate of Origin (DCO).
+
+# Explanation
+
+Most of the explanation of why we are doing this is recorded in the
+`radicle.community` links above. What this means is that all repositories under
+`radicle-dev` must comply with the following sections.
+
+# Recommendation
+
+In this section we include the templates that need to be included and also other
+recommendations to make the licensing experience as smooth as possible for both
+contributors and core developers.
+
+## License File
+
+Every repository MUST include a `LICENSE` file which corresponds to this
+[file][license-file].
+
+## DCO File
+
+Every repository MUST include a `DCO` which corresponds to this
+[file][dco-file].
+
+
+## CONTRIBUTING File
+
+Every repository MUST include a `CONTRIBUTING.md` file. The
+inclusion of the `CONTRIBUTING.md` file will mean that GitHub will prompt new
+contributors to look at the file before making a pull request.
+
+The file must include the following section:
+
+```
+## Certificate of Origin
+
+By contributing to this project, you agree to the [Developer Certificate of
+Origin (DCO)](./path/to/DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution.
+
+In order to show your agreement with the DCO you should include at the end of the commit message,
+the following line: Signed-off-by: John Doe <john.doe@example.com>, using your real name and email.
+
+This can be done easily using the -s flag on the git commit.
+
+If you find yourself pushed a few commits without Signed-off-by, you can still add it afterwards.
+```
+
+## DCO Bot
+
+TODO
+
+## Merging Pull Requests
+
+TODO
+
+# Open Questions
+
+* Should we include a PR template, or do we leave this up to the first time
+  contribution bot?
+* Can we disable the merge button on GitHub?
```

Comment:
No. https://github.com/isaacs/github/issues/5 Somebody posted there a workaround, that by protecting the branches with impossible rules one can effectively turn the green button of doom into a red button of nuclear wasteland.